Makefile

```diff
@@ -252,20 +252,26 @@ update-go-version:
 goversioncheck:
 	./verify-go-version.sh
 
+TRAY_RELEASE ?= packaging/tmp/crc-tray-macos.tar.gz
+
 packagedir: LDFLAGS+= -X '$(REPOPATH)/pkg/crc/version.macosInstallPath=$(MACOS_INSTALL_PATH)' $(RELEASE_VERSION_VARIABLES)
 packagedir: clean check_bundledir $(BUILD_DIR)/macos-amd64/crc $(HOST_BUILD_DIR)/crc-embedder
+	echo -n $(CRC_VERSION) > packaging/VERSION
 	sed -e 's/__VERSION__/'$(CRC_VERSION)'/g' -e 's@__INSTALL_PATH__@$(MACOS_INSTALL_PATH)@g' packaging/darwin/Distribution.in >packaging/darwin/Distribution
 	sed -e 's/__VERSION__/'$(CRC_VERSION)'/g' -e 's@__INSTALL_PATH__@$(MACOS_INSTALL_PATH)@g' packaging/darwin/welcome.html.in >packaging/darwin/Resources/welcome.html
 	sed -e 's/__VERSION__/'$(CRC_VERSION)'/g' -e 's@__INSTALL_PATH__@$(MACOS_INSTALL_PATH)@g' packaging/darwin/postinstall.in >packaging/darwin/scripts/postinstall
 	chmod 755 packaging/darwin/scripts/postinstall
-	mkdir -p packaging/root/"$(MACOS_INSTALL_PATH)"/$(CRC_VERSION)/
-	$(HOST_BUILD_DIR)/crc-embedder download packaging/root/"$(MACOS_INSTALL_PATH)"/$(CRC_VERSION)/
+	mkdir -p packaging/tmp/
+	$(HOST_BUILD_DIR)/crc-embedder download packaging/tmp/
+
+	mkdir -p packaging/root/Applications
+	tar -C packaging/root/Applications -xvzf $(TRAY_RELEASE)
```

Comment:
Doesn't `xvzf` should be only `vzf`? here we are compressing `packaging/root/Applications` to `TRAY_RELEASE` and `-x` is for extract.

Reply:
No, here we are uncompressing the tray in `packaging/root/Applications `. crc setup won't have to uncompress it at setup time.
The application will be directly ready in /Applications/CodeReady Containers.app right after pkg installation.

Reply:
Ack, I was confuse with `-C` flag and thought it is `-c` :)